app/services/jwt_token_validate.rb

```diff
@@ -0,0 +1,71 @@
+require 'jwt'
+require 'rest-client'
```

Comment:
Rails should load these automatically. 😕

Reply:
removed them, thanks :)